Attendee.test.js

```diff
@@ -0,0 +1,31 @@
+import { Attendee } from './Attendee';
+
+test('Check for correct value of screenName', () => {
+    const expectedScreenName = "Jaz";
+    const expectedId = "JTK614";
+    const expectedTimePolled = "2 minutes ago";
+    const attendee = new Attendee(expectedScreenName, expectedId, expectedTimePolled);
+    const gotScreenName = attendee.getScreenName();
+    expect(gotScreenName).toBe(expectedScreenName);
```

Comment:
Just test the rest of the getters here. Instead of having a separate test function for each getter. Update the test description after